showcase/src/app/demo-ui/pages/date-range-picker/demo-date-range-picker-page.component.html

```diff
@@ -22,5 +22,59 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 		</div>
 		<mat-checkbox [(ngModel)]="isDisabled">isDisabled</mat-checkbox>
 	</example-viewer>
+
+	<example-viewer
+		[extensions]="['HTML', 'TS']"
+		filesPath="date-range-picker/date-mask"
+		exampleTitle="SHOWCASE.DEMO.DATE_RANGE_PICKER.DEFAULT_MASK"
+	>
+		<div>
+			<stark-date-range-picker
+				rangePickerId="date-mask-range-picker"
+				rangePickerName="date-mask-range-picker"
+				[isDisabled]="isDisabled"
```

Comment:
The `isDisabled` property is not included in the TS example file of this date-range-picker...